openbas-api/src/main/java/io/openbas/atomic_testing/AtomicTestingUtils.java

```diff
@@ -2,15 +2,20 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import io.openbas.atomic_testing.AtomicTestingMapper.ExpectationResultsByType;
-import io.openbas.atomic_testing.AtomicTestingMapper.InjectTargetWithResult;
 import io.openbas.atomic_testing.AtomicTestingMapper.ResultDistribution;
+import io.openbas.atomic_testing.form.InjectTargetWithResult;
 import io.openbas.database.model.Inject;
 import io.openbas.database.model.InjectExpectation;
 import io.openbas.database.model.InjectExpectation.EXPECTATION_TYPE;
-import org.jetbrains.annotations.NotNull;
-
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.OptionalDouble;
 import java.util.stream.Collectors;
+import org.jetbrains.annotations.NotNull;
```

Comment:
is it a problem to use org.jetbrains.annotations NotNull instead of jakarta.validation.constraints one ?

Reply:
Thank you! I'll make the change :). In fact, I prefer using Jakarta or javax :)